<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_registrar_aluno">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Registrar Aluno</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_registrar_aula">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Registrar Aula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_visualizar_registros">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Visualizar Registros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_instrumentos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Adicionar Instrumentos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Menu Principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
